static/sass/_snapcraft_navigation.scss

```diff
@@ -0,0 +1,10 @@
+@mixin snapcraft-navigation {
+  .snapcraft-navigation {
```

Comment:
Please just add styles to `p-navigation` instead of creating a new class.

Reply:
done